doc/Language/operators.pod6

```diff
@@ -2075,7 +2055,7 @@ X<Numeric equality operator>.
 Coerces both arguments to L<Numeric|/type/Numeric> (if necessary); returns C<True>
 if they are equal.
 
-=head2 infix C«!=»
+=head2 infix C«!=», infix C«≠»
```

Comment:
OK, I see what's being done here, using a coherent header. The problem is that I'm not sure the initial ones were correct. I'll have to check the whole indexing thing to see what we have, but we might have to change the whole thing. =head2 indexes by default if it finds an "infix", so we need to see what happens here actually.